packages/components/src/components/post-togglebutton/post-togglebutton.scss

```diff
@@ -6,7 +6,13 @@
   cursor: pointer;
   outline-offset: tokens.get('focus-outline-offset', helpers.$post-focus) !important;
   outline: tokens.get('focus-outline-color', helpers.$post-focus) none
-  tokens.get('focus-outline-width', helpers.$post-focus) !important;
+    tokens.get('focus-outline-width', helpers.$post-focus) !important;
+
+  @include utilities.high-contrast-mode() {
```

Comment:
The hover styles are missing for the toggle button. But actually, this only happens in the header as the `post-togglebutton` should otherwise be used with a `.btn` class. 

Can you move this styles to the `packages/styles/src/components/header/index.scss` file?